README.md

```diff
@@ -1,99 +1,68 @@
 # PTF Analysis
 
-This is the codebase for analysing the ROOT files produced from the (PMT test facility) PTF. It handles the loading of files, accessing the data (e.g. PMT waveform samples and Phidget readings), fitting of waveforms to produce a tree of fitted parameters, and analyses of the waveform fits (including charge, timing, and efficiency measurements).
-
-## Table of Contents
-
-### 1. [Directory Layout](#directory_layout)
-### 2. [Installation](#installation)
-### 3. [Getting the PTF data](#data)
-### 4. [Usage](#usage)
-
-## Directory Layout <a id="directory_layout"></a>
-
-```bash
-.
-+-- bin/                                # Location of compiled executables
-+-- include/                            # Header files
-+-- macros/                             # ROOT macros to produce plots from the output of the analysis executables
-+-- magnetic-field/                     # Python scripts to process the output of field_to_csv
-+-- obj/                                # Location for compiled .o files
-+-- ptf_bfield/                         # Standalone analysis of the predicted PTF magnetic field
-+-- src/                                # Source files
-+-- Makefile                            # Makefile to build executables
-+-- field_to_csv.cpp
-+-- ptf.config.dat                      # A configuration file that sets analysis options
-+-- ptf_analysis.cpp
-+-- ptf_charge_analysis.cpp
-+-- ptf_field_analysis.cpp
-+-- ptf_qe_analysis.cpp
-+-- ptf_timing_analysis.cpp
-+-- ptf_ttree_analysis.cpp
-```
+This is a simple C++ library for reading the ROOT files produced from the PTF. It handles loading the files and provides a simple way to access the data.
```

Comment:
Seems like a big change to the README. Did you mean to remove all the top-level descriptions?

Reply:
Yes, probably a mistake, has to get back to the right version